src/Web/Suavemente/Input.hs

```diff
@@ -105,6 +106,15 @@ checkbox label = mkInput parseJSON $ \name v ->
         <input id="{name}" onchange="onChangeFunc(event)" type="checkbox" {bool ("" :: String) "checked='checked'" v} autocomplete="off">
         </td></tr>|]
 
+------------------------------------------------------------------------------
+-- | Create an input driven by an HTML checkbox without table tags.
+checkbox_ :: String -> Bool -> Suave Bool
```

Comment:
what do this and `textbox_` buy you?

Reply:
Just the use of div as a collection or wrapper of inputs.  It looks like <div> can't be used to wrap a <tr>.  See https://stackoverflow.com/questions/23440362/can-we-add-div-inside-table-above-every-tr/23440419. You have to use tbody instead, it seems, but this felt like lock-in to inputs always being one big table.

If the tr and td tags were removed from the input html, you would probably need to keep labels and inputs separate, so formatting like <tr><td>label</td><td>input</td></tr> could eventually be injected. `tableize` for the existing method.